FHEM/00_SIGNALduino.pm

```diff
@@ -3150,20 +3150,20 @@ sub SIGNALduino_Attr(@) {
   ## Change rfmode
   elsif ($aName eq 'rfmode')          # change receive mode
   {
-    if (not grep /$aVal/, @rfmode) {
-      $hash->{logMethod}->($name, 1, "$name: Attr, $aName $aVal is not supported");
-      return 'ERROR: The rfmode is not supported';
-    }
-
-    if ($init_done) {
-      my $ret = main::SIGNALduino_Attr_rfmode($hash,$aVal);
-
-      if (defined $ret) {
-        return $ret;
-      } else {
-        $hash->{logMethod}->($name, 3, "$name: Attr, $aName switched to $aVal");
+    if( $cmd eq 'set' ) {
+      if (not grep /$aVal/, @rfmode) {
+        $hash->{logMethod}->($name, 1, "$name: Attr, $aName $aVal is not supported");
+        return 'ERROR: The rfmode is not supported';
       }
-    }
+      if ($init_done) {
```

Comment:
Hier stimmt die Einrückung nicht ;)

Reply:
In der Zeile konnte ich keine Fehler sehen, dafür waren aber in Zeile 3166 noch Tabs statt Leerzeichen :-)